requirements.txt

```diff
@@ -1,12 +1,15 @@
 # Requirements for asl-ml-immersion repository
-google-cloud-aiplatform==1.43.0
-google-cloud-pipeline-components==2.8.0
+
+# Google Cloud
+google-genai==0.2.2 # tentative version
+google-api-core==2.24.0
+google-cloud-storage==2.19.0
```

Comment:
Added `google-api-core` and `google-cloud-storage` explicitly so the installation process doesn't need to install multiple versions and check compatibilities.